pkg/provider/aws/plugin/metadata/funcs.go

```diff
@@ -135,9 +135,19 @@ func cfn(clients AWSClients, name string) (interface{}, error) {
 		parameters[*p.ParameterKey] = p
 	}
 
+	// index outputs by name
+	outputs := map[string]interface{}{}
+	for _, p := range output.Stacks[0].Outputs {
+		if p.OutputKey == nil {
+			continue
+		}
+		outputs[*p.OutputKey] = p
+	}
+
```

Comment:
I tried to test locally, but not sure how to test, any suggestions would be helpful.

Reply:
The only way I can see is to mock the API call but tbh it really isn't testing much given how simple the code fragment is.

Reply:
Thanks